src/serializers/computed_fields.rs

```diff
@@ -52,8 +52,12 @@ impl ComputedFields {
             // Do not serialize computed fields
             return Ok(());
         }
-        for computed_fields in &self.0 {
-            computed_fields.to_python(model, output_dict, filter, include, exclude, extra)?;
+        for computed_field in &self.0 {
+            let field_extra = Extra {
+                field_name: Some(computed_field.property_name.as_str()),
```

Comment:
Why do we not do any alias logic here?

Reply:
there should be a mapping `match extra.by_alias` in the `.to_python()`